sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.supporting-users.controller.ts

```diff
@@ -184,7 +184,13 @@ export class SupportingUserSupportingUsersController extends BaseController {
       supportingUserType,
       application.programYear,
     );
-
+    // Ensure the offering intensity provided is the same from the application.
+    if (
+      payload.offeringIntensity !==
+      application.currentAssessment.offering.offeringIntensity
+    ) {
+      throw new UnprocessableEntityException("Invalid offering intensity.");
```

Comment:
Please add the new exception to the Swagger decorator `ApiUnprocessableEntityResponse`.